src/super_gradients/recipes/cityscapes_kd_base.yaml

```diff
@@ -1,7 +1,7 @@
 # Distillation for semantic segmentation on Cityscapes dataset.
 #
 # Instructions:
-#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
+#   0. Make sure that the data is stored in dataset_params.dataset_dir or add "+dataset_params.data_dir=<PATH-TO-DATASET>" at the end of the command below (feel free to check ReadMe)
```

Comment:
I think this is not correct, but not in the way this PR attempts to fix it.
A + sign in hydra override is for cases where you want to override a setting parameter that is not in config. Unless you add + hydra will raise an exception, which is actually good, because it indicates this parameter is missing.
Here the instruction is wrong about `data_dir` - https://github.com/Deci-AI/super-gradients/blob/f7f02294a0663bbe13d313ae2afd6a7af0181eef/src/super_gradients/recipes/dataset_params/cityscapes_dataset_params.yaml#L2 . It should be `root_dir`.

Reply:
@hakuryuu96 small fix here, otherwise LGTM

Reply:
I've checked locally, and here's what I got

_dataset_params.root_dir_ -- doesn't work
_train_dataset_params.root_dir_ -- doesn't work
_dataset_params.train_dataset_params.root_dir=... dataset_params.val_dataset_params.root_dir=..._ -- worked 

@BloodAxe is this expected or am I missing something?